R/load_epidata.R

```diff
@@ -24,26 +24,27 @@
 #' load_epidata_raw(pathogen = "marburg", table = "outbreak")
 #' @export
 load_epidata_raw <- function(pathogen, table = c("article", "parameter",
-                                                "outbreak", "model")) {
-  
+                                                "outbreak", "model", "par_types")) {
```

Comment:
Would be call this variable with a more helpful name. I think param_type sounds better to me than par_types? Regardless, we need to make it singular rather than plural to be consistent with other arguments.